flink-ai-flow/ai_flow_plugins/scheduler_plugins/airflow/airflow_scheduler.py

```diff
@@ -319,15 +292,17 @@ def get_job(self, job_name: Text, execution_id: Text) -> Optional[JobInfo]:
                 return None
             else:
                 project_name, workflow_name = self.dag_id_to_namespace_workflow(dag_run.dag_id)
-                return JobInfo(job_name=job_name,
-                               state=self.airflow_state_to_state(task.state),
-                               workflow_execution
-                               =WorkflowExecutionInfo(workflow_info=WorkflowInfo(namespace=project_name,
-                                                                                 workflow_name=workflow_name),
-                                                      execution_id=dag_run.run_id,
-                                                      state=self.airflow_state_to_state(dag_run.state)))
+                return [JobExecutionInfo(job_name=job_name,
```

Comment:
The `job_execution_id` of the `JobExecutionInfo` doesn't set the value with task.try_number.